static/app/views/relocation/getStarted.tsx

```diff
@@ -0,0 +1,146 @@
+import {useContext, useState} from 'react';
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+
+import {Button} from 'sentry/components/button';
+import SelectControl from 'sentry/components/forms/controls/selectControl';
+import Input from 'sentry/components/input';
+import {RelocationOnboardingContext} from 'sentry/components/onboarding/relocationOnboardingContext';
+import {t} from 'sentry/locale';
+import ConfigStore from 'sentry/stores/configStore';
+import {space} from 'sentry/styles/space';
+import testableTransition from 'sentry/utils/testableTransition';
+import StepHeading from 'sentry/views/relocation/components/stepHeading';
+
+import {StepProps} from './types';
+
+function GetStarted(props: StepProps) {
+  const regions = ConfigStore.get('regions');
+  const [region, setRegion] = useState(regions[0].name);
+  const [orgSlugs, setOrgSlugs] = useState('');
+  const relocationOnboardingContext = useContext(RelocationOnboardingContext);
+
+  const handleContinue = (event: any) => {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    relocationOnboardingContext.setData({formData});
+    props.onComplete();
+  };
+  return (
+    <Wrapper>
+      <StepHeading step={1}>{t('Basic Information Needed to Get Started')}</StepHeading>
```

Comment:
Do we use sentence case for our section titles? It's what we use in the onboarding flow, so I think so? If so, this should be:

```suggestion
      <StepHeading step={1}>{t('Basic information to get started')}</StepHeading>
```

Reply:
I just used the case style used in the figma mockups, let's confirm with @jmanhart

Reply:
If anything let's stick to one style and just ship, then change in the future if need be